Screenbox/Pages/HomePage.xaml

```diff
@@ -23,37 +23,29 @@
                 <ResourceDictionary Source="/Controls/Templates/GridViewItemTemplates.xaml" />
             </ResourceDictionary.MergedDictionaries>
 
-            <XamlUICommand
-                x:Key="PlayCommand"
-                Command="{x:Bind ViewModel.PlayCommand}"
-                IconSource="{ui:SymbolIconSource Symbol=Play}"
-                Label="{strings:Resources Key=Play}" />
-
-            <XamlUICommand
-                x:Key="PlayNextCommand"
-                Command="{x:Bind ViewModel.PlayNextCommand}"
-                IconSource="{ui:FontIconSource FontFamily={StaticResource ScreenboxSymbolThemeFontFamily},
-                                               Glyph=&#xF5EB;}"
-                Label="{strings:Resources Key=PlayNext}" />
-
-            <XamlUICommand
-                x:Key="RemoveCommand"
-                Command="{x:Bind ViewModel.RemoveCommand}"
-                IconSource="{ui:SymbolIconSource Symbol=Clear}"
-                Label="{strings:Resources Key=Remove}" />
-
-            <XamlUICommand
-                x:Key="PropertiesCommand"
-                Command="{StaticResource ShowPropertiesCommand}"
-                IconSource="{ui:FontIconSource Glyph=&#xE946;}"
-                Label="{strings:Resources Key=Properties}" />
```

Comment:
To give some context, the XamlUICommands are there to be shared between context menu items and future command bar buttons, like the inbox media player. It's fine to remove these for now.

Reply:
Didn't knew that, but wouldn't it be better for sharing to simply extract the entire MenuFlyout?

Reply:
What do you mean by extracting the entire menu Flyout? XamlUICommand includes command, icon, label, and keyboard accelerator. Having controls share the same XamlUICommand will ensure they behave the same, functionally and aesthetically.

Reply:
Like [this](https://github.com/videolan/vlc-winrt/blob/master/app/VLC.Universal/Views/UserControls/Flyouts/AlbumMenuFlyout.xaml)

Reply:
I'm confused. How would you map these MenuFlyoutItems into Buttons or AppBarButtons?

Reply:
Oh maybe [this](https://github.com/videolan/vlc-winrt/blob/master/app/VLC.Universal/Resources/Flyouts.xaml)? There must be a way